cluster-provision/k8s/1.17/manifests/cnao/operator.yaml

```diff
@@ -153,9 +153,9 @@ spec:
         - name: KUBEMACPOOL_IMAGE
           value: quay.io/kubevirt/kubemacpool:v0.8.3
         - name: MACVTAP_CNI_IMAGE
-          value: quay.io/kubevirt/macvtap-cni:v0.1.0
+          value: quay.io/kubevirt/macvtap-cni:v0.2.0
         - name: OPERATOR_IMAGE
-          value: quay.io/kubevirt/cluster-network-addons-operator:0.32.0
+          value: quay.io/kubevirt/cluster-network-addons-operator:0.35.0
```

Comment:
not sure 0.35.0 is a good version (I see 0.35.1, for instance): https://github.com/kubevirt/cluster-network-addons-operator/commits/master

Reply:
I feel you have just replaced values in the yaml directly. Didn't you?

What about bumping to 0.38.0 right away? This PR should not be concerned about macvtap CNI, but about CNAO in the right version.

Reply:
You should make sure to consume released manifests from a certain version. By injecting random changes in the operator yaml may result in mismatch. Like the one you just introduced with the operator image still set to 0.32.0 on line 176

Reply:
> I feel you have just replaced values in the yaml directly. Didn't you?

Guilty as charged; is it automated ?... If so, how to do it ? 

> 
> What about bumping to 0.38.0 right away? This PR should not be concerned about macvtap CNI, but about CNAO in the right version.

I couldn't care less about whatever is new on 0.35.0 onwards. Having said that, it is exactly the same amount of effort; I can do it.

Reply:
Similar PR by Quique: https://github.com/kubevirt/kubevirtci/pull/312/files

I think you have to do it manually, I can't find a script for it.

Find matching files in https://github.com/kubevirt/cluster-network-addons-operator/releases/tag/0.38.0 and use them as replacement.

Reply:
Yeah, let's first then this to work, then let's decide to which version should we bump this.